.github/actions/setup_env/Makefile

```diff
@@ -8,7 +8,7 @@ install_fasm2bels:
 	cd ${BFASST_PATH_FASM2BELS} && make build
 	cd ${BFASST_PATH_FASM2BELS} && make test-py
 
-	cd ../../.. && python scripts/run_design.py designs/basic/and3/ xilinx_and_reversed
+	cd ../../.. && python scripts/run.py VivadoBitAnalysis designs/basic/and3
```

Comment:
Hmm, the VivadoBitAnalysis actually uses the PhysicalToLogical tool, which is private.  The CI has access to it, but the public does not.  Can you make a new flow `VivadoBitToNetlist` that stops after Xray, and use that here?

Jeff

Reply:
Do we need to add tests to this new flow? @jgoeders